db/init.sql

```diff
@@ -28,6 +28,7 @@ CREATE TABLE lists_users (
   id integer GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
   user_id integer NOT NULL REFERENCES users ON DELETE CASCADE,
   list_id integer NOT NULL REFERENCES lists ON DELETE CASCADE
```

Comment:
Syntax error here is causing the db initialization to fail. We need to add a comma at the end of line 30.

Reply:
Updated